MetaMask/metamask-mobile
# feat(ramp): add NetworkSwitcher view

**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/coding_guidelines/CODING_GUIDELINES.md)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add the appropiate QA label when dev review is completed
    - `needs-qa`: PR requires manual QA.
    - `No QA/E2E only`: PR does not require any manual QA effort. Prior to merging, ensure that you have successful end-to-end test runs in Bitrise. 
    - `Spot check on release build`: PR does not require feature QA but needs non-automated verification. In the description section, provide test scenarios. Add screenshots, and or recordings of what was tested.
5. Add `QA Passed` label when QA has signed off (Only required if the PR was labeled with `needs-qa`)
6. Add your team's label, i.e. label starting with `team-` (or `external-contributor` label if your not a MetaMask employee)

**Description**

This PR adds a new screen for the ramp flows that handles the case the current selected network is not supported

**Screenshots/Recordings**


https://github.com/MetaMask/metamask-mobile/assets/1024246/fa2a138d-dfc3-453f-ab0c-6cd0f86a821d

**QA**
Open the buy flow using a deeplink (metamask://buy-crypto) or universal link (https://metamask.app.link/buy-crypto) in an unsupported network to see the new screen. 

Here's a dummy url with the links https://62hv4l.csb.app/

**Issue**

fixes #???

**Checklist**
